lib/debezium/types.go

```diff
@@ -130,6 +131,27 @@ func (f Field) ParseValue(value any) (any, error) {
 		return f.DecodeDecimal(bytes)
 	case KafkaVariableNumericType:
 		return f.DecodeDebeziumVariableDecimal(value)
+	case DateTimeWithTimezone:
+		dtString, isOk := value.(string)
+		if !isOk {
+			return nil, fmt.Errorf("expected string got '%v' with type %T", value, value)
+		}
+
+		// We don't need to pass `additionalDateFormats` because this data type layout is standardized by Debezium
+		extTime, err := ext.ParseExtendedDateTime(dtString, nil)
+		if err == nil {
+			return extTime, nil
+		}
+
+		// Check if the year exceeds 9999, or is negative
+		if parts := strings.Split(dtString, "-"); len(parts) == 3 {
```

Comment:
If the year is negative there will be four parts, the first being an empty string.

Reply:
True, added a guardrail above